user-manual/getting-started/getting-started.rst

```diff
@@ -162,8 +162,10 @@ encouraged to use a third-party analytics tool - for more information, see:
 .. TIP::
 
    A :term:`developer` who wishes to hide the Popular this week menu from
-   view can do so by commenting out Line 23 in
-   ``/apps/qubit/modules/staticpage/templates/homeSuccess.php``. 
+   view can do so by commenting out lines 29-32 in
```

Comment:
This looks good! However since BS2 is deprecated but not removed as of now, I would suggest adding a `NOTE` section that explains that the aforementioned instructions apply for BS5 themes, and share the older instructions that were meant for BS2 themes in the `NOTE` as well.

Reply:
Thanks for the comment - I updated the text accordingly, let me know what you think.